src/main/osd/osd_elements.c

```diff
@@ -177,6 +178,8 @@ static unsigned activeOsdElementCount = 0;
 static uint8_t activeOsdElementArray[OSD_ITEM_COUNT];
 static bool backgroundLayerSupported = false;
 
+extern float rMat[3][3];
```

Comment:
Never declare `extern` in .c file, include `imu.h` instead.
This could lead to very difficult bugs in variable type is changed.